build.xml

```diff
@@ -0,0 +1,67 @@
+<project name="chloris" default="build" basedir=".">
+    <!-- SUITE -->
+    <target name="build" depends="fixtures" />
+    <target name="test" depends="behat" />
+    <!-- END -->
+    <!-- COMPOSER -->
+    <target name="composer">
```

Comment:
add "retry" phing task to avoid errors if first attempt to download composer fails